packages/derived-wallet-solana/src/signAptosTransaction.ts

```diff
@@ -36,32 +36,26 @@ export async function signAptosTransactionWithSolana(input: SignAptosTransaction
     solanaPublicKey,
     rawTransaction,
     signingMessageDigest,
+    domain
   });
 
   const response = await wrapSolanaUserResponse(solanaWallet.signIn!(siwsInput));
-
   return mapUserResponse(response, (output): AccountAuthenticator => {
     if (output.signatureType && output.signatureType !== 'ed25519') {
       throw new Error('Unsupported signature type');
     }
-
     // The wallet might change some of the fields in the SIWS input, so we
     // might need to include the finalized input in the signature.
     // For now, we can assume the input is unchanged.
-
     const signature = new Ed25519Signature(output.signature);
 
-    const serializer = new Serializer();
-    serializer.serialize(signature);
-    serializer.serialize(input.rawTransaction.rawTransaction);
-    serializer.serializeVector(input.rawTransaction.secondarySignerAddresses ?? []);
-    serializer.serializeOption(input.rawTransaction.feePayerAddress);
-    const authenticator = serializer.toUint8Array();
+    const accountIdentity = `${solanaPublicKey.toBase58()}${domain}`
```

Comment:
As mentioned in a previous comment, this logic is already present in `SolanaDerivedPublicKey`.
How about extract it into a function and use that in both places?
```
function getSolanaDerivedAccountIdentity(solanaPublicKey, domain): Uint8Array {
  return new TextEncoder(`${solanaPublicKey.toBase58()}${domain}`);
}
```